language/tools/move-cli/README.md

```diff
@@ -130,33 +194,126 @@ fun main(account: &signer) {
 }
 ```
 
-This script invokes the `publish` function of our `Test` module, which will publish a resource of type `Test::Resource` under the signer's account:
+This script invokes the `publish` function of our `Test` module, which will
+publish a resource of type `Test::Resource` under the signer's account:
 
 ```
-λ move run script.move --signers 0xf -c
-Compiling 1 user module(s)
+λ move run test_script.move --signers 0xf -v
+Compiling transaction script...
 Changed resource(s) under 1 address(es):
-  Changed 1 resource(s) under address 0000000000000000000000000000000f:
-    Added type 00000000::Test::Resource: [U64(10)]
-Committed changes.
+  Changed 1 resource(s) under address 0000000000000000000000000000000F:
+      Added type 0x2::Test::Resource: [U64(10)]
 ```
 
 We can also inspect this newly published resource using `move view`:
 
 ```
-λ move view storage/0x0000000000000000000000000000000f/resources/0x00000000000000000000000000000002\:\:Test\:\:Resource
-resource 00000000::Test::Resource {
-    i: 10
+λ move view storage/0x0000000000000000000000000000000F/resources/0x00000000000000000000000000000002::Test::Resource.lcs
+resource 0x2::Test::Resource {
+        i: 10
 }
 ```
 
+### Cleaning state
 
-## Using the CLI with Libra modules and genesis state
+Since state persists from one call to the Move CLI to another, there will
+frequently be times where you want to start again at a clean state.  This can
+be done using the `move clean` command. This command will remove all previously
+published modules and resources from the state, and will start you back at a
+fresh state empty state.
 
-Take a look at `tests/testsuite/liba_smoke/args.txt`. This test uses the CLI to run a fairly realistic Libra genesis setup and a few basic transactions. Running
+```
+λ move clean
+λ move view storage/0x0000000000000000000000000000000F/resources/0x00000000000000000000000000000002::Test::Resource.lcs
+Error: `move view <file>` must point to a valid file under storage
+```
+
+## Testing with the Move CLI
+
+The Move CLI also has a built-in testing framework. Each test is run
+independently in its own sandbox so state does not persist from one test to
+another, or from one test run to another.
+
+Each test is structured as a directory consisting of an `args.txt` file that
+specifies a sequence of Move CLI commands that should be run in that directory.
+Usually, there are two subdirectories, one for `modules`, and the other for
```

Comment:
The project structure section would let us kill this sentence.

Reply:
I'll add